tests/unit/test_merlion_handler.py

```diff
@@ -0,0 +1,220 @@
+import os
+import time
+from unittest.mock import patch
+import pandas as pd
+
+from mindsdb_sql import parse_sql
+
+# How to run:
+#  env PYTHONPATH=./ pytest tests/unit/test_merlion_handler.py
+# Warning: uncomment and run too many sql-testcase in detector and forecaster will cost a long time
+from mindsdb.integrations.handlers.merlion_handler.adapters import DefaultForecasterAdapter, SarimaForecasterAdapter, \
+    ProphetForecasterAdapter, MSESForecasterAdapter, IsolationForestDetectorAdapter, \
+    WindStatsDetectorAdapter, ProphetDetectorAdapter
+# from mindsdb.integrations.handlers.merlion_handler.adapters import DefaultDetectorAdapter
+from .executor_test_base import BaseExecutorTest
+
+import pandas as pd
+from pathlib import Path
+import requests
+
+
+class TestMerlion(BaseExecutorTest):
```

Comment:
Tests are working for you? For me they fail by some reason (need to debug to figure out).

Reply:
hey， please use the command "PYTHONPATH=./ pytest tests/unit/test_merlion_handler.py", at first, please change current path to ./mindsdb/. Here I should apologize that I‘ve written the mistaken command in comment as "PYTHONPATH=./ pytest tests/unit/test_ml_handlers.py"， the right command should be "PYTHONPATH=./ pytest tests/unit/test_merlion_handler.py".

Reply:
All the four test cases can pass in this way.

Reply:
Problem was brought to the pr in the latest comits... I will fix it, sorry...

Reply:
Fixed

Reply:
Yes, now tests are working. 
Thanks!

Reply:
my pleasure